core/src/main/java/com/opentable/logging/CommonLogFields.java

```diff
@@ -86,18 +76,24 @@ default String getOtEnvFlavor() {
     @JsonProperty("severity")
     String getSeverity();
 
-    @JsonProperty("logmessage")
+    @JsonProperty("message")
     String getMessage();
 
-    @JsonProperty("threadname")
+    @JsonProperty("thread-name")
     String getThreadName();
 
-    @JsonProperty("throwable")
+    @JsonProperty("exception")
     String getThrowable();
 
     /** Written by the encoder, value is ignored for serialization. */
-    @JsonProperty("sequencenumber")
+    @JsonProperty("sequence-number")
     default long getSequenceNumber() {
         return Long.MIN_VALUE;
     }
+
+    /** Declare the OTL type for selectorless processing. */
+    @JsonProperty("@loglov3-otl")
+    default String getLoglov3Otl() {
```

Comment:
That makes this technically an API breaking change but I tend to agree that it's worth that cost.